docs/coordinates/solarsystem.rst

```diff
@@ -5,47 +5,91 @@
-    This functionality requires that the 
-    `jplephem <https://pypi.python.org/pypi/jplephem>`_ package
-    is installed. This is most easily acheived via ``pip install jplephem``,
-    although whatever package management system you use might have it as well.
+   To use JPL ephemerides requires that the `jplephem
+   <https://pypi.python.org/pypi/jplephem>`_ package is installed. This is
+   most easily achieved via ``pip install jplephem``, although whatever
+   package management system you use might have it as well.
 
-Three functions are provided; :meth:`~astropy.coordinates.get_body`, 
-:meth:`~astropy.coordinates.get_moon` and 
-:meth:`~astropy.coordinates.get_body_barycentric`. The first
-two functions return |SkyCoord| objects in the `~astropy.coordinates.GCRS` frame,
-whilst the latter returns a `~astropy.coordinates.CartesianRepresentation` of the barycentric position
-of a body (i.e in the `~astropy.coordinates.ICRS` frame).
+Three functions are provided; :meth:`~astropy.coordinates.get_body`,
+:meth:`~astropy.coordinates.get_moon` and
+:meth:`~astropy.coordinates.get_body_barycentric`. The first two functions
+return |SkyCoord| objects in the `~astropy.coordinates.GCRS` frame, whilst the
+latter returns a `~astropy.coordinates.CartesianRepresentation` of the
+barycentric position of a body (i.e in the `~astropy.coordinates.ICRS` frame).
 
-Here are some examples of these functions in use::
+Here are some examples of these functions in use with the ``de432s``
+ephemeris (which is a smaller, ~10 MB, file valid between 1950 and 2050)::
+
+  >>> from astropy.time import Time
+  >>> from astropy.coordinates import solar_system_ephemeris, EarthLocation  
+  >>> from astropy.coordinates import get_body_barycentric, get_body, get_moon
+  >>> t = Time("2014-09-22 23:22")
+  >>> loc = EarthLocation.of_site('greenwich')
+  >>> solar_system_ephemeris.set('de432s') # doctest: +REMOTE_DATA, +IGNORE_OUTPUT
+  <ScienceState solar_system_ephemeris: 'de432s'>
+  >>> get_body('jupiter', t, loc) # doctest: +REMOTE_DATA, +FLOAT_CMP
+  <SkyCoord (GCRS: obstime=2014-09-22 23:22:00.000, obsgeoloc=[ 3949481.6898847   -550931.9118969   4961151.73733443] m, obsgeovel=[  40.1745933   288.00078051   -0.        ] m / s): (ra, dec, distance) in (deg, deg, km)
+      (136.90234781, 17.03160686, 889196019.15383542)>
+  >>> get_moon(t, loc) # doctest: +REMOTE_DATA, +FLOAT_CMP
+  <SkyCoord (GCRS: obstime=2014-09-22 23:22:00.000, obsgeoloc=[ 3949481.6898847   -550931.9118969   4961151.73733443] m, obsgeovel=[  40.1745933   288.00078051   -0.        ] m / s): (ra, dec, distance) in (deg, deg, km)
+      (165.51840735, 2.32900633, 407226.68749643)>
+  >>> get_body_barycentric('moon', t) # doctest: +REMOTE_DATA, +FLOAT_CMP
+  <CartesianRepresentation (x, y, z) in km
+      (150107535.1073409, -866789.11996916, -418963.55218495)>
+
+For lower precision estimates that do not require downloading an ephemeris::
+
+  >>> solar_system_ephemeris.set('approximate')
```

Comment:
maybe for this one use the `with` form of science state?  I.e. ,
```
>>> with solar_system_ephemeris.set('approximate'):
>>> .... jup = get_body('jupiter', t, loc)  # doctest: +FLOAT_CMP
>>> jup
<SkyCoord (GCRS ...
```
that shows that you can do it in the context-manager way (which I consider "safer", psersonally)